src/App.js

```diff
@@ -387,14 +619,32 @@ function Stat({
           fColor={"#a93226"}
           increased={deadCountIncrease}
         >
-          Deaths
+        <HtmlTooltip
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Deaths</Typography>
+              <em>{"All cases that resulted in death, mostly found in the elderly."}</em>
+            </React.Fragment>
+          }
+        >
+          <Button>Deaths</Button>
+        </HtmlTooltip>
         </Tag>
         <Tag
           number={curedCount}
           fColor={"#00b321"}
           increased={curedCountIncrease}
         >
-          Recovered
+        <HtmlTooltip
+          title={
+            <React.Fragment>
+              <Typography color="inherit">Recovered</Typography>
+              <em>{"All cases that are no longer infected."}</em>
```

Comment:
Good place to elaborate on what has to happen for a patient to be deemed 'recovered'.

Reply:
Tbh I'm not actually sure about its definition as well haha. What do you think it could be?